spec/models/arrival_spec.rb

```diff
@@ -50,7 +50,7 @@
   end
 
   describe '#convert_nil_to_blank' do
-    it '空文字をnilに変換すること' do
+    it 'nilを空文字に変換すること' do
       arrival.memo = nil
       arrival.save!
       expect(arrival.memo == '').to be true
```

Comment:
@kutimiti1234
PRありがとうございます！間違い直してもらえて、とても嬉しいです☺️

`expect`で変な書き方をしていたので、ついでに修正お願いできたりしますでしょうか？？🙇‍♀️
```suggestion
      expect(arrival.memo).to eq ''
```